src/convertToObject.js

```diff
@@ -3,16 +3,31 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+  const styleProperties = sourceString
+    .replace(/;/g, '').split('\n')
+    .filter(i => i.includes(':')).map(i => i.split(':')
+      .map(j => j.trim()).join(': '));
```

Comment:
better to move each of the method call to a new line, because the way you have it now is totally not readable.
```suggestion
  const styleProperties = sourceString
    .replace(/;/g, '')
    .split('\n')
    .filter(i => i.includes(':'))
    .map(i => i.split(':')
      .map(j => j.trim())
      .join(': '));
```